content/en/docs/quick-start/create-workspace-and-project.md

```diff
@@ -168,38 +168,39 @@ If you want to expose services using the type `LoadBalancer`, you need to use th
 
-1. Log in to the console as `admin` again and go to **Access Control**.
+1. Log in to the KubeSphere web console as `admin` again and go to **Access Control**. 
 
-2. In **Account Roles**, there are four system roles listed which cannot be deleted or edited. Click **Create** and set a **Role Identifier**. In this example, a role named `roles-manager` will be created.
-
-   ![customized-role](/images/docs/quickstart/create-workspaces-projects-accounts/customized-role.png)
+2. Click **Account Roles** on the left navigation bar and **Create** on the right.
 
    {{< notice note >}}
-It is recommended you enter a description for the role as it explains what the role is used for. The role created here will be responsible for role management only, including adding and deleting roles.
-   {{</ notice >}}
 
-   Click **Edit Authorization** to continue.
+   The preset roles on the **Account Roles** page cannot be edited and deleted.
 
-3. In **Access Control**, select the authorization that you want this role to contain. For example, **Users View**, **Roles Management** and **Roles View** are selected for this role. Click **OK** to finish.
+   {{</ notice >}}
 
-   ![edit-authorization](/images/docs/quickstart/create-workspaces-projects-accounts/edit-authorization.png)
+3. In the **Create Account Role** dialog box, set the name (for example, `clusters-admin`), alias, and description of the role, and click **Edit Permissions**.
 
    {{< notice note >}}
-   **Depend on** means the major authorization (the one listed after **Depend on**) needs to be selected first so that the affiliated authorization can be assigned.
-   {{</ notice >}}
-
-4. Newly-created roles will be listed in **Account Roles**. You can click the three dots on the right to edit it.
 
-   ![roles-manager](/images/docs/quickstart/create-workspaces-projects-accounts/roles-manager.png)
+   This example demonstrates how to create a role responsible for cluster management.
 
-5. In **Accounts**, you can add a new account and grant it the role `roles-manager` or change the role of an existing account to `roles-manager` by editing it.
+   {{</ notice >}}
 
-   ![edit-role](/images/docs/quickstart/create-workspaces-projects-accounts/edit-role.png)
+4. In the **Edit Permissions** dialog box, set the role permissions (for example, select **Cluster Management**) and click **OK**.
 
    {{< notice note >}}
-   The role of `roles-manager` overlaps with `users-manager` while the latter is also capable of user management. This example is only for demonstration purposes. You can create customized roles based on your needs.
+
+   * Some permissions depend on other permissions. The dependency is specified by the **Depends on** field under each permission.
+   * When a permission is selected, the permission it depends on is automatically selected.
+   * To deselect a permission, you need to deselect its subordinate permissions first.
+
```

Comment:
Add another sentence in notes:

In this example, the role `clusters-admin` contains the permissions **Cluster Management** and **Cluster Viewing**.